docs/protocol/discovery-node/plugins/network_monitoring.md

```diff
@@ -0,0 +1,160 @@
+---
+sidebar_label: Network Monitoring
+sidebar_position: 1
+---
+
+# Audius Network Monitoring
+
+**Network Monitoring** refers to network-wide metrics we collect from a discovery node and aggregate to assess the health of the entire network.
+
+Note - This is different from node-level metrics which are scraped by prometheus from the content and discovery nodes through a special `/prometheus_metrics` route.
+
+Here are some metrics that we intend to:
+
+- The number of CID on each CN that have been replicated at least once
+- The number of CID on each CN that have ***NOT*** been replicated at least once
+- [ex 1] **The number of users with a specific CN as their primary**
+- The number of users with a specific CN in their replica set
+- CID replication across the CNs
+- CID replication factor
+- The number of users with their data syncs across 0, 1, 2, or 3 CNs
+
+Metrics are collected and calculated on a nightly basis.
+
+---
+
+## Installation
+
+### Prereqs
+
+- Docker installed
+- Direct access to a Discovery Node postgres DB
+- Secrets file, with the correct permission, formatted like so
+    
+    ```json
+    {
+    	"signatureSpID": 42,
+    	"signatureSPDelegatePrivateKey": "0x123123123123123"
+    }
+    ```
+    
+
+### Running services
+
+By default, this will run every service which includes
+
+- network monitoring CRON
+- postgres
+- prometheus push-gateway
+- prometheus
+- grafana
+
+```bash
+docker-compose up
```

Comment:
Where do I put my json secrets file? I think it feels like steps missing here.

Reply:
The path to the secrets file can be configured in env vars. I should add a section on env vars